.github/workflows/ci.yml

```
@@ -31,11 +31,12 @@ jobs:
           key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
           restore-keys: |
             ${{ runner.os }}-test-${{ env.cache-name }}-
-            ${{ runner.os }}-test-
+            ${{ runner.o }}-test-
             ${{ runner.os }}-
       - run: julia --project=@. -e '
             ENV["JULIA_PKG_SERVER"] = "";
             using Pkg;
+            Pkg.develop(Widgets);
```

```suggestion
            Pkg.add(url="https://github.com/piever/Widgets.jl", rev="master")
```

I think this is the correct way to get it to load the master branch from a given URL

---

This doesn't work on 1.0